### DISCUSSION TITLE
게임 지원 산업 관련 정리

### DISCUSSION THREAD

**Discussion by fkdl0048** [GG-Studio-990001/Grayed-Game#35] [General]
- [경기게임오디션](https://www.play4apply.com/home/reg/step2.php)
  - 2024년 기준 ~4/15(월)까지 접수
- [SGM](https://careers.smilegate.com/student/membership)
  - 매년 3~5월 지원 가능
  - 6개월 활동 지원 (금액, 창작 지원, 등등 매우 다양)
- [방구석 인디게임쇼](https://www.bigameshow.com/bigs2023/)
  - 다른 공모전에 비해 좀 더 가벼운 느낌?
  - 현재 게임에 대한 홍보용으로 적합
- [G-Star 온라인 공모전](https://indie.onstove.com/ko/store/G-STAR2023)
  - G-star 선발 관련 공모전
- [BIC](https://www.bicfest.org/)
  - 인디게임 페스티벌, 가장 유명
  - 접수 4월 말~6월 중순
- [PlayX4](https://www.playx4.or.kr/)
   - 접수 4월, 행사 5월
- [인디고(IndieGo)](https://futurelab.center/front/program/program-view?G_MENU_SEQ=&contSeq=696)
  - 3~4월
  - 스마일게이트 주관 인디게임 창작 공모전
  - 22년도, 23년도 진행
- [인디크래프트](https://www.snip.or.kr/indiecraft/main/index.do)
  - 접수 3월
- [GIGDC](https://www.gigdc.or.kr/main.php)
  - 접수 6월
- [MWU 유니티코리아 어워즈](https://unitysquare.co.kr/madewith/mwu)
  - 접수 7월
  - 매번 이벤트로 유니티 굿즈 뿌려서 일단 지원하면 무조건 이득 ^_^
- [Out Of Index](https://www.outofindex.org/)
  - [텀블벅 링크](https://tumblbug.com/ooi2022)
  - 실험적인 게임을 소개하는 페스티벌이라 우리 게임에 어울릴지도?
  - 행사 12월
- [WASD 인디게임 공모전](https://forcreators.stoveindie.com/wasd2024)
  - 아마 2024년에 처음
  - 접수 7월 말~8월
  - 예선, 본선, 결선 존재

**Comment by wooyn730** [comment]
제가 알고 있는 것도 조금 남겨봅니다~ (안겹치는 것만)
날짜는 정확하지 않을 수 있습니다

_(내용 본문으로 이동함)_

**Reply by fkdl0048** [reply]
가독성 때문에 위 본문으로 올려도 될까요?

**Reply by wooyn730** [reply]
지금 봤네요 넴넴

**Comment by wooyn730** [comment]
https://forcreators.stoveindie.com/post/festival/2024

**Comment by wooyn730** [comment]
[[스토브 소식] 2025 게임 콘텐츠 맞춤형 지원사업 안내 (~3/20, 17시)](https://forcreators.stoveindie.com/post/?bmode=view&idx=156605289)
